generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -24,39 +22,59 @@ stages:
     - release
     - deploy
 
-<%_ if (buildTool == 'gradle') { _%>
+<%_ if (buildTool === 'gradle') { _%>
 before_script:
     - export GRADLE_USER_HOME=`pwd`/.gradle
+    - chmod +x gradlew
```

Comment:
In private gitlab latest version it does work. I haven't tested it in gitlab.com

But I can see it is being used in most of the projects. So let's have it .